<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Тестовое задание</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>781</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>   Удалять входные файлы?</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>   Операция</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_Mask"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>   Путь для сохранения результата</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QRadioButton" name="radioButton_OR">
       <property name="text">
        <string>OR</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>   Папка поиска</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QWidget" name="widget_3" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QToolButton" name="toolButton_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>10</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>481</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 184, 184);</string>
        </property>
        <property name="text">
         <string>Выберете папку для поиска</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QWidget" name="widget_5" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QRadioButton" name="radioButton_Repl">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Заменять</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_AddN">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>190</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавлять цифру в конце</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QRadioButton" name="radioButton_XOR">
       <property name="text">
        <string>XOR</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>   Периодичность</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>   Таймер</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>мин.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>   Действия </string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_Value"/>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>10</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>481</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 184, 184);</string>
        </property>
        <property name="text">
         <string>Выберете папку для результата</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>    Значение переменной</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QLineEdit" name="lineEdit_Period">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>0</y>
          <width>371</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QRadioButton" name="radioButton_AND">
       <property name="text">
        <string>AND</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>   Маска входных файлов</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QWidget" name="widget_4" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QRadioButton" name="radioButton_Del">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Да</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_NoDel">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>0</y>
          <width>190</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Нет</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="9" column="1" colspan="3">
      <widget class="QWidget" name="widget_6" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QRadioButton" name="radioButton_Per">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Да</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_NoPer">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>190</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Нет (остановка)</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>465</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Модификация входных файлов бинарной операцией</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Тестовая программа</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
